### PR TITLE
os/board/rtl8721csm: change of BLE adv log API

### PR DESCRIPTION
**Pull request by yeetee179**
- change of log printing api from dbg to debug_print, to reduce default log printing when starting and stopping BLE adv

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@yeetee179 Could you leave logs before and after in the commit description?

**Comment by yeetee179** [comment]
I have modified the commit, the logs are not removed, but replaced with debug_print,

**Review by sunghan-chang — COMMENTED** [review]
It's not a modification or removal of BLE log. It's change of usage of log printing api.
Please leave why and what difference is.

**Comment by yeetee179** [comment]
commit title and message is updated accordingly

**Comment by sunghan-chang** [comment]
> I have modified the commit, the logs are not removed, but replaced with debug_print,



> @yeetee179 Could you leave logs before and after in the commit description?

@yeetee179 Could you leave before and after?

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c [hunk 1]
```diff
@@ -485,18 +485,18 @@ int ble_tizenrt_scatternet_handle_upstream_msg(uint16_t subtype, void *pdata)
...
         {
             ret = le_adv_start();
-            if(GAP_CAUSE_SUCCESS == ret)
-                dbg("[Upstream] Start Adv Success \n");
-            else
-                dbg("[Upstream] Start Adv Fail !! 0x%x \n", ret);
+            if(GAP_CAUSE_SUCCESS == ret)
+                debug_print("[Upstream] Start Adv Success \n");
+            else
+                debug_print("[Upstream] Start Adv Fail !! 0x%x \n", ret);
```
Comment (giwon-nam) [inline os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c:491]:
```suggestion
                dbg("[Upstream] Start Adv Fail !! 0x%x \n", ret);

```

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c [hunk 2]
```diff
@@ -485,18 +485,18 @@ int ble_tizenrt_scatternet_handle_upstream_msg(uint16_t subtype, void *pdata)
...
 		case BLE_TIZENRT_MSG_STOP_ADV:
 			ret = le_adv_stop();
-			if(GAP_CAUSE_SUCCESS == ret)
-                           dbg("[Upstream] Stop Adv Success \n");
-                       else
-                           dbg("[Upstream] Stop Adv Fail !! 0x%x \n", ret);
+            if(GAP_CAUSE_SUCCESS == ret)
+                debug_print("[Upstream] Stop Adv Success \n");
+            else
+                debug_print("[Upstream] Stop Adv Fail !! 0x%x \n", ret);
```
Comment (giwon-nam) [inline os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c:499]:
```suggestion
                dbg("[Upstream] Stop Adv Fail !! 0x%x \n", ret);

```

**Review by giwon-nam — COMMENTED** [review]
I think it would be better to keep the fail log printed with dbg.

**Comment by sunghan-chang** [comment]
> I think it would be better to keep the fail log printed with dbg.

Could you let us know why dbg is better than debug_print?

**Comment by yeetee179** [comment]
@giwon-nam I have modified it as suggested.

**Review by giwon-nam — APPROVED** [review]

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c [hunk 3]
```diff
@@ -699,12 +699,12 @@ void ble_tizenrt_scatternet_app_handle_dev_state_evt(T_GAP_DEV_STATE new_state,
...
-                    new_state.gap_scan_state,
-                    new_state.gap_conn_state,
-                    new_state.gap_adv_state,
-                    cause);
+    debug_print("init state  %d, scan state %d, conn_state %d, adv_state %d, cause 0x%x \n",
+                    new_state.gap_init_state,
+                    new_state.gap_scan_state,
+                    new_state.gap_conn_state,
+                    new_state.gap_adv_state,
+                    cause);
```
Comment (giwon-nam) [inline os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c:707]:
```suggestion
    dbg("init state  %d, scan state %d, conn_state %d, adv_state %d, cause 0x%x \n",
                    new_state.gap_init_state,
                    new_state.gap_scan_state,
                    new_state.gap_conn_state,
                    new_state.gap_adv_state,
                    cause);
```

**Review by giwon-nam — COMMENTED** [review]
We decided to keep the code that prints `adv_state` and delete the adv start and stop logs. Can we change the below line to use debug_print?
Line [738](https://github.com/Samsung/TizenRT/pull/6458/files#diff-ccf0c29b7db3d30d65d9b4af616e0289ed8ebe24ba382b2d9fb44c80466d82e8R738), [744](https://github.com/Samsung/TizenRT/pull/6458/files#diff-ccf0c29b7db3d30d65d9b4af616e0289ed8ebe24ba382b2d9fb44c80466d82e8R744)

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c (hunk 3 above)
Comment (yeetee179) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c]: @giwon-nam the modification is made accordingly

**Review by hs36-kim — APPROVED** [review]
LGTM

**Pull request merged**